express/blocks/pricing/pricing.js

```diff
@@ -75,29 +82,114 @@ function decorateHeader($block, header) {
   animateHeader($header, words);
 }
 
-function buildPlanDropdown() {
-  const $dropdown = createTag('select', { class: 'plan-dropdown' });
-  const $option1 = createTag('option', { value: 'Option 1' });
-  $option1.innerHTML = 'Option 1';
-  $dropdown.append($option1);
-  return $dropdown;
+function getPlanOptions(planTitle, planOptions) {
+  const options = [];
+  planOptions.forEach((option) => {
+    const optionPlan = option['Option Plan'];
+    if (planTitle === optionPlan) {
+      options.push(option);
+    }
+  });
+  return options;
 }
 
-function decoratePlans($block, plans) {
+function replaceUrlParam(url, paramName, paramValue) {
+  const parameterValue = paramValue == null ? '' : paramValue;
```

Comment:
I would recommend sanitizing the `paramValue` before processing it as it can be user-provided and potentially dangerous.

Reply:
This is fixed with the latest commit. Thank you for the suggestion.